### PR TITLE
clean up active error handling

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: forch/faucet_state_collector.py [hunk 1]
```diff
@@ -557,13 +557,15 @@ def get_switch_egress_path(self, src_switch, src_port=None):
     def get_host_path(self, src_mac, dst_mac, to_egress):
         """Given two MAC addresses in the core network, find the active path between them"""
         if not src_mac:
-            return {'error': 'Empty eth_src. Please use list_hosts to get a list of hosts'}
+            return self._make_summary(State.error,
```
Comment (didovesei) [inline forch/faucet_state_collector.py]: It seems that State.error is not defined in shared_constants.proto

File: proto/switch_state.proto [hunk 2]
```diff
@@ -98,7 +98,7 @@ message SwitchState {
     Attributes attributes = 1;
 
     // Port state
-    string port_state = 2;
+    State.State port_state = 2;
```
Comment (didovesei) [inline proto/switch_state.proto]: Do we need to replace port state with State in faucet_state_collector.py?

Comment (didovesei) [reply proto/switch_state.proto]: E.g. faucet_state_collector.py line 410

port_map[PORT_STATE] = STATE_UP if port_up else STATE_DOWN

File: forch/faucet_state_collector.py (hunk 1 above)
Comment (grafnu) [reply forch/faucet_state_collector.py]: changed to State.broken

File: proto/switch_state.proto (hunk 2 above)
Comment (grafnu) [reply proto/switch_state.proto]: I didn't seem like it... I think it just effects the json conversion, but didn't seem required for internal operation.  If there's a specific spot/change you're thinking of, let me know.

**Comment by grafnu** [comment]
port_map is not a proto, it's the old-style... so it needs to be handled
with old-style states, I think.

**Pull request merged**